weepickle/src/com/rallyhealth/weepickle/v1/EnumFromTos.scala

```diff
@@ -0,0 +1,34 @@
+package com.rallyhealth.weepickle.v1
+
+import java.util.concurrent.ConcurrentHashMap
+
+import com.rallyhealth.weepickle.v1.core.Types
+import com.rallyhealth.weepickle.v1.implicits.{DefaultFroms, DefaultTos}
+
+trait EnumFromTos extends Types with DefaultTos with DefaultFroms {
```

Comment:
Downside of a separate trait is that I can't add any new methods to it without breaking binary compat. One new trait per PR isn't sustainable.

I'm considering just dumping these into the abstract class.